client/src/utils/BGraphQueryUtil.ts

```diff
@@ -147,4 +148,133 @@ export function bioNodeOutDegree (bgraphPathQuery, clause: Filter): any {
     // Document type is an edge
     return false;
   });
-}
+};
+
+// EPI PATH QUERIES
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const epiPathPre = (bgraphPathQuery, clause: Filter): any => {
+  return bgraphPathQuery.filter(document => {
+    if (document._type === 'node') {
+      const node = document;
+      return clause.values[0] === node.label;
+    }
+    // Document type is an edge
+    return false;
+  })
+    .start()
+    .unique();
+};
+// eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/explicit-module-boundary-types
+const epiPathPreNodeFilterLoop = (bgraphPathQuery, _clause: Filter): any => {
+  return bgraphPathQuery.in(); // TODO: Why are edge directions backward??
+};
+// eslint-disable-next-line @typescript-eslint/no-unused-vars, @typescript-eslint/explicit-module-boundary-types
+const epiPathPreEdgeFilterLoop = (bgraphPathQuery, _clause: Filter): any => {
+  return bgraphPathQuery.in(); // TODO: Why are edge directions backward??
+};
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const epiPathPost = (bgraphPathQuery, clause: Filter): any => {
+  return bgraphPathQuery.suspend(document => {
+    if (document._type === 'node') {
+      const node = document;
+      return clause.values[1] === node.label;
+    }
+    // Document type is an edge
+    return false;
+  })
+    .repeat(15) // TODO: Implement cycle detection in bgraph to remove epi path length limit
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    .filter(_ => false)
+    .unsuspend();
+};
+
+// Add filter terms here to change the order in which they are processed in bgraph
+export const EDGE_PRIORITY_RANK = 2;
+export const NODE_PRIORITY_RANK = 4;
+export const PATH_PRIORITY_RANK = 7;
+export const filterTermToPriorityRank = {
+  // BIO Edge Terms
+  [QUERY_FIELDS_MAP.BIO_EDGE_PRE.field]: 1,
```

Comment:
I guess if we want to be consistent then these constants should be GRAPH (for BIO) and MODEL (for EPI)

Reply:
I've made the change to the environment file. However, I think we should create a task for the larger naming change as it goes past the scope of what has been modified here.